christin/index.js

```diff
@@ -0,0 +1,55 @@
+//1. Print Exponential Values
+//Write a function that accepts two numbers and validate that they are numbers.
+//After that, the function should print y exponential values starting from x.
+
+/*
+- For example if we have function(3, 5) The function should print 3 9 27 81 243. Prints 5 exponential values of 3.
+- function(2, 8) The function prints 2 4 8 16 32 64 128 256. Prints 8 exponential values of 2.*/
+
+function getExponents(number, value) {
+  let box = [];
+  let zahl = 1;
+  //if (number === typeof Number && value === typeof Number) {
+  for (let i = 1; i <= value; i++) {
+    let res = (zahl *= number);
+    box.push(res);
+    //Math.pow(number, value)
+  }
+  console.log(box);
+}
+getExponents(2, 8);
+
+/*2. Fruits
+Create a function named . Declare a variable outside of the function and store your favorite fruit as a value. Reassign the variable within the function and print "My favorite fruit is: x".*/
+
+let fruits = ["mango", "coconut", "pineapple", "cucumber"];
+
+// option 1
+
+function printFavoriteFruit() {
+  console.log(`My favorite fruit is ${fruits[0]}.`);
```

Comment:
Task 2: Please reassign the variable, means you change the value of fruits variable within the function and then try to print.

Reply:
allright, done :) sorry for the missunderstanding

Reply:
No prob:) Thanks for rework:))